meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -314,6 +314,10 @@ RESIN_CONFIGS[xtables] = " \
     "
 
 # Deactivate the audit susbsystem and the audit syscall
+RESIN_CONFIGS_DEPS[audit] = " \
+    CONFIG_HAVE_AUDITSYSCALL=n \
+    CONIFG_SECURITY=n \
```

Comment:
@agherzan @floion extract from the Kconfig
Symbol: AUDIT [=y]
Type  : boolean
Prompt: Auditing support
Depends on: NET [=y]
Selected by: SECURITY_APPARMOR [=y] && SECURITY [=y] && NET [=y]